lib/dry/cli/banner.rb

```diff
@@ -104,7 +105,7 @@ def self.extended_command_arguments(command)
       # rubocop:disable Metrics/AbcSize
       def self.extended_command_options(command)
         result = command.options.map do |option|
-          name = Hanami::Utils::String.dasherize(option.name)
+          name = Dry::Inflector.new.dasherize(option.name)
```

Comment:
It'd be better to make inflector configurable or at least as a `Dry::CLI::Inflector` constant.